build_tools/cmake/build_tracy_capture.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Build Tracy capture tool for different platforms. This is separated from
+# building benchmark tools becuase capture tool doesn't always run on the target
+# becnhmark devices (e.g. recording tracy via port forwarding).
+
+set -euo pipefail
+
+BUILD_DIR="${1:-${IREE_TARGET_BUILD_DIR:-build-tracy}}"
+PLATFORM="${IREE_TARGET_PLATFORM:-linux}"
+ARCH="${IREE_TARGET_ARCH:-x86_64}"
+
+PLATFORM_ARCH="${PLATFORM}-${ARCH}"
+
+source build_tools/cmake/setup_build.sh
+# source build_tools/cmake/setup_ccache.sh
+
+declare -a args
+args=(
+  "-G" "Ninja"
+  "-B" "${BUILD_DIR}"
+  "-DPython3_EXECUTABLE=${IREE_PYTHON3_EXECUTABLE}"
+  "-DPYTHON_EXECUTABLE=${IREE_PYTHON3_EXECUTABLE}"
+  "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+  "-DIREE_BUILD_COMPILER=OFF"
+  "-DIREE_BUILD_SAMPLES=OFF"
+  "-DIREE_BUILD_TESTS=OFF"
+  "-DIREE_BUILD_TRACY=ON"
+)
+
+case "${PLATFORM_ARCH}" in
+  linux-x86_64)
+    # Assumes the builder is Linux x86_64.
+    ;;
+  *)
+    echo "Building tracy capture for ${PLATFORM_ARCH} not supported yet"
+    exit 1
```

Comment:
Tracy already provides build configurations... I'd really rather we not invest any more in our own build configuration
https://github.com/wolfpld/tracy/tree/master/profiler/build (read the Tracy manual)

Reply:
Removed.